product_attribute_types/README.rst

```diff
@@ -0,0 +1,34 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product attribute types
+=======================
+
+This module extends product attributes, adds a new type range, and also the
+option of defining if it is required and a default value for the attribute.
+
```

Comment:
I would state here that this module provides basic functionality that is extended upon by more functional modules.